includes/class-amp-theme-support.php

```diff
@@ -992,6 +992,8 @@ static function( $html ) {
 		add_filter( 'cancel_comment_reply_link', [ __CLASS__, 'filter_cancel_comment_reply_link' ], 10, 3 );
 		add_action( 'comment_form', [ __CLASS__, 'amend_comment_form' ], 100 );
 		remove_action( 'comment_form', 'wp_comment_form_unfiltered_html_nonce' );
+		add_filter( 'get_comments_link', [ __CLASS__, 'amend_comments_link' ] );
+		add_filter( 'respond_link', [ __CLASS__, 'amend_comments_link' ] );
```

Comment:
When looking at how Twenty Twenty links to comments, I noticed an unfamiliar function: `comments_popup_link()`. It turns out that this function applies `respond_link` filters on the URL instead of calling `get_comments_link()` when there are no comments. While our classic Reader mode templates aren't using this function, someone may be using this function in their custom Reader mode templates.